apps/sms/test/marionette/conversation_test.js

```diff
@@ -2,7 +2,7 @@
 'use strict';
 
 var assert = require('chai').assert;
-
+var ThreadMock = require('./mocks/mock_message_thread');
```

Comment:
`libs` doesn't seem the right place to me, you're right, we need to put this generator away from mocks. How about `test/marionette/generators`?